src/css/styles.css

```diff
@@ -2,6 +2,124 @@
 @import './global.css';
 
 body {
-  font-family: 'Nunito', sans-serif; /* Using Nunito font family*/
-  background-color: var(--primary-color); /*using a css variable*/
+  background-color: var(--light-blue-color);
+}
+
+.meals {
+  max-width: 774px;
+  margin: auto;
+}
+
+.title {
```

Comment:
I think that in this selector we could remove the height, font-weight and line-height. With the font-size and margin we make sure that it takes the necessary space and the font-weight:400 is the default value.